e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -124,9 +144,7 @@ Scenario('Gatekeeper submits final version of standard directions', async (I, ca
     config.administrationActions.amendRespondents,
     config.administrationActions.amendOther,
     config.administrationActions.amendInternationalElement,
-    config.administrationActions.amendOtherProceedings,
     config.administrationActions.amendAttendingHearing,
-    config.applicationActions.uploadDocuments,
   ]);
```

Comment:
Didn't realise that these checks exist. Do you think it would be sensible to have a test that checks the dropdowns contains the expected actions?